src/screens/ContactScreen/index.js

```diff
@@ -239,6 +240,8 @@ const ContactScreen = ({navigation}) => {
             isCreateCommunity: true,
             topic: topicCommunityName
           });
+        } else {
+          SimpleToast.show(response?.message, SimpleToast.SHORT);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You are using `SimpleToast.show()` to display error messages. While this is fine for development, it might not be the best user experience in production. Consider implementing a more user-friendly way to handle and display errors.

```diff
- SimpleToast.show(response?.message, SimpleToast.SHORT);
+ // Suggested change: Implement a user-friendly error handling mechanism
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->